App crashes on startup on some Samsung phones

No interesting stack trace, but appears to only happen on Samsung phones:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3792)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3968)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2307)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:246)
  at android.app.ActivityThread.main (ActivityThread.java:8512)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:602)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
```

[aTox-short.log](https://github.com/evilcorpltd/aTox/files/8053947/aTox-short.log)

Done, I guess this is the key:

```02-11 22:42:48.007 20767 20767 E AndroidRuntime: Caused by: java.lang.SecurityException: ltd.evilcorp.atox was not granted  this permission: android.permission.WRITE_SETTINGS.```

--- a/atox/src/main/kotlin/ui/NotificationHelper.kt
+++ b/atox/src/main/kotlin/ui/NotificationHelper.kt
@@ -69,7 +69,9 @@ class NotificationHelper @Inject constructor(
             .setName(context.getString(R.string.friend_requests))
             .build()
 
-        val ringtone = RingtoneManager.getActualDefaultRingtoneUri(context, RingtoneManager.TYPE_RINGTONE)
+        // We can't use getActualDefaultRingtoneUri as Samsung requires the WRITE_SETTINGS permission for that. :D
+        // See: https://github.com/evilcorpltd/aTox/issues/958
+        val ringtone = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_RINGTONE)
         val callChannelBuilder = NotificationChannelCompat.Builder(CALL, NotificationManagerCompat.IMPORTANCE_HIGH)
             .setName(context.getString(R.string.calls))
             .setVibrationEnabled(true)

